src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,135 @@
+package duke;
```

Comment:
Should your class files be in a folder "duke", in lower cases instead?

Reply:
Yea, for some reason even after changing the file name the file name keeps getting reset to Duke...

Reply:
Hmm, you should look into this... Or just manually rename the folder in your file browser.